mods/parsoid.js

```diff
@@ -530,6 +551,45 @@ function cheapBodyInnerHTML(html) {
     }
 }
 
+/**
+ * Replaces sections in original content with sections provided in sectionsJson
+ */
+function replaceSections(original, sectionsJson) {
+    var sectionOffsets = original['data-parsoid'].body.sectionOffsets,
+        newBody = cheapBodyInnerHTML(original.html.body);
+    Object.keys(sectionsJson)
+    .sort(function(id1, id2) {
```

Comment:
That's correct :+1: But, please, let's not confuse the reader / maintainer of the code - use `offset1` for `id1` (idem for `offset2`) in the line below.